doc/mvf_v4.rst

```diff
@@ -225,36 +225,42 @@ Calibration solutions
 
 Streams of type ``sdp.cal`` have the following keys.
 
-``antlist`` (list of string)
+``antlist`` (list of string, length ``n_ants``)
     List of antenna names. Arrays of calibration solutions use this
     order along the antenna axis.
 
-``pol_ordering`` (list of string)
+``pol_ordering`` (list of string, length ``n_pols``)
     List of polarisations (from ``v`` and ``h``). Arrays of calibration
```

Comment:
Just curious, are the polarisations hard-coded to be h & v?  Can they be  R and L or more than 2  eg. IQUV?

Reply:
I suppose in theory if we ever build a circular-polarisation dish they could be R&L, but that's not defined for now and I don't expect software to handle such values.

These are for inputs, not correlation products, so I don't think more than two or Stokes parameters makes any sense.